docs/getting-started/products/ycrv/overview.md

```diff
@@ -26,11 +26,10 @@ Yearn passes all benefits of its veCRV position on to yCRV users who hold one of
 
 - [**st-yCRV (Staking Rewards):**](#staked-ycrv) [yVault](https://medium.com/iearn/yearn-finance-explained-what-are-vaults-and-strategies-96970560432) that receives admin fees and bribes from locked CRV.
 - [**lp-yCRV (Liquidity Pool Rewards):**](#lpd-ycrvcrv) yVault for CRV/yCRV LP tokens, autocompounds emissions and fees.
-- [**vl-yCRV (Curve Gauges Voting Power):**](#voting-with-ycrv) Vote power mechanism to vote on Curve gauge weights.
```

Comment:
maybe worth adding a note here like,
"Note: a third activated token, vl-yCRV, was never rolled out due to lack of demand."